pipeline/stop_places/sql/stop_places.sql

```diff
@@ -575,12 +627,19 @@ CREATE OR REPLACE VIEW stop_area_elements AS (
 
 /*
  * Final site path link view
- * Currently this is just a wrapper of the "path_links" table.
- * TODO: JOIN "path_links" with "paths_elements_ref" and "highways" GROUP BY "path_id" and somehow aggregate tags
+ * The tables "paths_elements_ref" and "highways" are joined to create a view that contains all path links with their osm tags.
+ * Only one element of the "paths_elements_ref" table is joined to be able to later generate the xml field "accessFeatureType".
+ * There should be no case where an access feature (stairs, ...) is composed of multiple OSM elements.
  */
 CREATE OR REPLACE VIEW final_site_path_links AS (
-  SELECT stop_area_relation_id AS relation_id, path_id::text as id, '{}'::jsonb AS tags, geom, start_node_id as "from", end_node_id as "to"
-  FROM path_links
+  SELECT
+    stop_area_relation_id AS relation_id, pl.path_id::text as id, highways.tags as tags, pl.geom, start_node_id as "from", end_node_id as "to"
+    FROM path_links pl
+  JOIN 
+  (SELECT DISTINCT ON (path_id) * FROM paths_elements_ref) per
+    ON per.path_id = pl.path_id 
+  LEFT JOIN highways
+    ON (highways.osm_id = per.osm_id AND highways.osm_type = per.osm_type)
 );
```

Comment:
In my tests this doesn't return the same amount of elements as `SELECT * FROM path_links`

The reason is because you use `JOIN` (which is the same as `INNER JOIN`) instead of `LEFT JOIN` on `path_links` and `paths_elements_ref`. This may throw away path_links that do not have a corresponding OSM element.

My suggestion is to use:
```SQL
-- use distinct to filter any duplicated joined paths
 SELECT DISTINCT ON (pl.path_id)
    -- fallback to empty tags if no matching element exists
    stop_area_relation_id AS relation_id, pl.path_id::text as id, COALESCE(highways.tags, '{}'::jsonb) as tags, pl.geom, start_node_id as "from", end_node_id as "to"
  FROM path_links pl
  LEFT JOIN paths_elements_ref per
    ON per.path_id = pl.path_id 
  LEFT JOIN highways
    ON highways.osm_id = per.osm_id AND highways.osm_type = per.osm_type
```

To be honest, I would expect that your INNER JOIN still works since the reference table should contain at least one element for every path_link unless you are (rightfully) excluding generated path links, which is probably what is happening?

Reply:
> To be honest, I would expect that your INNER JOIN still works since the reference table should contain at least one element for every path_link unless you are (rightfully) excluding generated path links, which is probably what is happening?

No path links are excluded on purpose, but some paths between quays (mapped as areas) are only consisting of short paths without any osm information. So no mapping between path link and highways can be made.

**Examples:** 

de:14511:30062:8:14 <-> de:14511:30062:8:13
```
"edges": [
                {
                    "edge_type": "footway",
                    "street_type": "none",
                    "crossing_type": "none",
                    "side": "center",
                    "osm_way_id": 0,
                    "from_node_osm_id": 0,
                    "to_node_osm_id": 0,
                    "name": "",
                    "marked_crossing_detour": 0,
                    "distance": 5.184103423948295,
                    "duration": 3.240064639967684,
                    "accessibility": 0.0,
                    "path": [
                        [
                            12.931116,
                            50.839424
                        ],
                        [
                            12.9311843,
                            50.8394063
                        ]
                    ],
                    "elevation_up": 0,
                    "elevation_down": 0,
                    "incline_up": true,
                    "handrail": null,
                    "duration_penalty": 0.0,
                    "accessibility_penalty": 0.0,
                    "area": false,
                    "marked_crossing_detour": 0,
                    "level": 0.0
                },
                {
                    "edge_type": "footway",
                    "street_type": "none",
                    "crossing_type": "none",
                    "side": "center",
                    "osm_way_id": 0,
                    "from_node_osm_id": 0,
                    "to_node_osm_id": 0,
                    "name": "",
                    "marked_crossing_detour": 0,
                    "distance": 0.12394596933538328,
                    "duration": 0.07746623083461454,
                    "accessibility": 0.0,
                    "path": [
                        [
                            12.9311843,
                            50.8394063
                        ],
                        [
                            12.931186,
                            50.839406
                        ]
                    ],
                    "elevation_up": 0,
                    "elevation_down": 0,
                    "incline_up": true,
                    "handrail": null,
                    "duration_penalty": 0.0,
                    "accessibility_penalty": 0.0,
                    "area": false,
                    "marked_crossing_detour": 0,
                    "level": 0.0
                }
            ]
```

![Bildschirmfoto 2023-07-18 um 15 53 37](https://github.com/OPENER-next/osm2vdv462/assets/50618489/3f688918-d188-4519-897e-d1d291c86d1d)

---

With your suggestion now these paths are also included :) (commit [069b518](https://github.com/OPENER-next/osm2vdv462/pull/15/commits/069b518c51383fdc5b6f17c49fd9c08ec11f6bb9))